src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,416 @@
+
+        private void OnConnectionChanged(object s, AvrcpControlConnectionChangedEventArgs e)
+        {
+            if (e.RemoteAddress != _remoteAddress)
+            {
+                return;
+            }
+            if (_taskForConnection != null && !_taskForConnection.Task.IsCompleted)
+            {
+                if (e.IsConnected == true)
+                {
+                    _taskForConnection.SetResult(true);
+                    ConnStateChanged?.Invoke(this, e);
+                }
+                else
+                {
+                    _taskForConnection.SetException(BluetoothErrorFactory.CreateBluetoothException((int)BluetoothError.OperationFailed));
+                }
+                _taskForConnection = null;
+                return;
+            }
+            if (_taskForDisconnection != null && !_taskForDisconnection.Task.IsCompleted)
+            {
+                if (e.IsConnected == false)
+                {
+                    _taskForDisconnection.SetResult(true);
+                    ConnStateChanged?.Invoke(this, e);
+                }
+                else
+                {
+                    _taskForDisconnection.SetException(BluetoothErrorFactory.CreateBluetoothException((int)BluetoothError.OperationFailed));
+                }
+                _taskForDisconnection = null;
+            }
+        }
+
+        /// <summary>
+        /// The AvrcpControlConnectionChangedEventArgs event is invoked when the connection status of device is changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<AvrcpControlConnectionChangedEventArgs> ConnStateChanged;
+
+        /// <summary>
+        /// The PositionChangedEventArgs event is invoked when the play position of a track is changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<PositionChangedEventArgs> PositionChanged;
+
+        /// <summary>
+        /// The PlayStateChangedEventArgs event is invoked when the play state of a track gets changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<PlayStateChangedEventArgs> PlayStateChanged;
+
+        /// <summary>
+        /// The TrackInfoChangedEventArgs event is invoked when info of a track gets changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<TrackInfoChangedEventArgs> TrackInfoChanged;
+
+        /// <summary>
+        /// Asynchronously connects the remote device
+        /// </summary>
+        /// <param name="remoteAddress">Address of device to be connected</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method fails</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public Task ConnectAsync(string remoteAddress)
```

Comment:
After connectioned, if I try to ConnectAsync with another remote address, What is happend?
as a code, _remoteAddress will be updated with a new remote address

Reply:
New connection will be created based on another remote address after disconnected for previous connection.

Reply:
Can I expect ConnectionStateChanged event? about disconnection and connection

Reply:
As you expect, disconnection and connection event will be called in above case.
I removed the remoteAddress param in ConnectAsync() because BluetoothAvrcpControl instance is only one for each device. so, no need remoteAddress.